content/webapp/components/IIIFViewer/MainViewer.tsx

```diff
@@ -84,6 +122,8 @@ type ItemRendererProps = {
     errorHandler?: () => void;
     externalAccessService: TransformedAuthService | undefined;
     accessToken?: string;
+    restrictedService: AuthExternalService | undefined;
+    placeholderId: string | undefined;
```

Comment:
np; we use a mix of `?` and `| undefined` which could be aligned, very much a nitpick